designer/server/src/common/constants/scopes.js

```diff
@@ -0,0 +1,12 @@
+import config from '~/src/config.js'
+
+export const SCOPE_READ = 'read'
+export const SCOPE_WRITE = 'write'
+
+/**
+ * Maps a group a user is a member of to the scopes they have access to.
+ * @type {{[key: string]: Array<string>}}
```

Comment:
Just linking this new file to the old thread in https://github.com/DEFRA/forms-designer/pull/145#discussion_r1595071183

Reply:
Sorry, that's what I was getting it

If you do `Array<string>` or `string[]` the compiler never knows what scopes are valid

If you exactly type the list of scopes, then it'll be able to error when rename or remove one

Reply:
In fact, can remove the `@type` entirely

Reply:
That's a fair point - it's redundant! Removed.